client/lib/load-script/README.md

```diff
@@ -4,8 +4,9 @@ Async Script Loader
 This utility function allows us to use a standardized method of loading remote scripts and injecting them into the `<head>` of our document to bring external functionality into the app.
 
 ### Usage
-```js
-loadScript = require( 'lib/load-script' );
+```es6
+import loadScript from 'lib/load-script';
```

Comment:
[That is still used in three places](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=loadjQueryDependentScript)